pkg/lib/cockpit-components-dropdown.tsx

```diff
@@ -36,7 +35,14 @@ import { EllipsisVIcon } from '@patternfly/react-icons';
  * require a separator between DropdownItem's use PatternFly's Divivder
  * component.
  */
-export const KebabDropdown = ({ dropdownItems, position = "end", isDisabled = false, toggleButtonId, isOpen, setIsOpen, props }) => {
+export const KebabDropdown = ({ dropdownItems, position = "end", isDisabled = false, toggleButtonId, isOpen, setIsOpen, props } : {
+    dropdownItems: React.ReactNode,
+    position?: "start" | "end" | "right" | "left" | "center",
+    isDisabled?: boolean,
+    toggleButtonId?: string;
+    isOpen?: boolean, setIsOpen?: React.Dispatch<React.SetStateAction<boolean>>,
```

Comment:
These on the other hand make sense, as they are truly optional attributes (i.e. the internal handling of open-ness).